plugins/modules/system/puppet.py

```diff
@@ -176,7 +183,7 @@ def main():
             logdest=dict(type='str', default='stdout', choices=['all', 'stdout', 'syslog']),
             show_diff=dict(
```

Comment:
Would you mind adding this comment? This will help the next person wondering what exactly this does :)
```suggestion
            # The following is not related to Ansible's diff; see https://github.com/ansible-collections/community.general/pull/3980#issuecomment-1005666154
            show_diff=dict(
```